src/request/components/request-detail-panel-logging.tsx

```diff
@@ -36,15 +97,15 @@ export class Logging extends React.Component<ILoggingProps, {}> {
                         </div>
                     </div>
                     <br/>
-                    <table className='table table-bordered table-striped tab-content-item'>
+                    <table className='table table-bordered table-striped tab-content-item tab-logs-table'>
                         <thead>
                             <tr className='table-col-title-group'>
-                                <th width='5%'><span className='table-col-title'>Ordinal</span></th>
-                                <th width='10%'><span className='table-col-title'><FontAwesomeIcon path=''/>Level</span></th>
-                                <th width='40%'><span className='table-col-title'>Message</span></th>
+                                <th width='10%'><span className='table-col-title'>Ordinal</span></th>
+                                <th width='10%'><span className='table-col-title tab-logs-table-icon-column'><FontAwesomeIcon path=''/>Level</span></th>
+                                <th width='58%'><span className='table-col-title tab-logs-table-icon-column'><FontAwesomeIcon path=''/>Message</span></th>
                                 <th width='10%'><span className='table-col-title'>From Start</span></th>
                                 <th width='10%'><span className='table-col-title'>Duration</span></th>
-                                <th />
+                                <th width='2%' />
```

Comment:
@chetsangnil's design was for the last column in the table to be fixed-width, offset from the end of the table (i.e. not taking the remainder of the available width, which is quite often the style of VS (Windows) tables).  While I'd guess that we might be able to get the same effect via CSS, the simplest way seemed to be just adding a "dummy" column.